lib/data/multipleBackendGateway.js

```diff
@@ -225,10 +225,18 @@ const multipleBackendGateway = {
 
     abortMPU: (key, uploadId, location, bucketName, log, cb) => {
         const client = clients[location];
+        const skipDataDelete = true;
         if (client.clientType === 'azure') {
-            const skipDataDelete = true;
             return cb(null, skipDataDelete);
         }
+        if (client.clientType === 'gcp') {
+            return client.abortMPU(key, uploadId, bucketName, log, err => {
+                if (err) {
+                    return cb(err);
+                }
+                return cb(null, skipDataDelete);
```

Comment:
The conditional can be done here, correct?

Reply:
yes. will move the condition to reduce duplicate code